### DISCUSSION TITLE
A few Ideas to make the experience more polished

### DISCUSSION THREAD

**Discussion by mrprofessork** [SecureAI-Tools/SecureAI-Tools#89] [Ideas]
- There is no sign-out button - which makes changing users a little difficult for the average user
- The User role should not be able to make organizational changes 
- add a back button similar to the back button on the user settings page 
- the ability to add users one at a time, might be useful if you have a small team and have to add a single user

**Comment by JayNakrani** [comment]
Thanks for starting this discussion. These are all great ideas and I'll tackle them soon. 

> The User role should not be able to make organizational changes

The user role can only view organization settings but I may have missed something. Can you tell us where exactly a user role is able to make changes to an organization?

**Reply by mrprofessork** [reply]
You are correct, I jumped the gun on this one. The User role can only view.  Thanks for getting back to me on this!
